go.mod

```diff
@@ -21,3 +22,5 @@ require (
 	github.com/urfave/cli v1.22.1
 	go4.org v0.0.0-20190218023631-ce4c26f7be8e // indirect
 )
+
+replace github.com/containers/image/v5 => github.com/lumjjb/image/v5 v5.0.0-20191125184705-a298da5c535d
```

Comment:
Please drop the replace here and use commit `502848a1358b` for vendoring c/image/v5.